Jupyter/QuantBook.cs

```diff
@@ -488,12 +488,14 @@ private PyObject Indicator(IndicatorBase<IndicatorDataPoint> indicator, IEnumera
 
             selector = selector ?? (x => x.Value);
 
-            foreach (var bar in history)
+            foreach (var slice in history)
```

Comment:
smart -- we should use slices for everything, makes it explicit how we need to handle the various data types.